v22.1/restore.md

```diff
@@ -63,16 +63,16 @@ You can control `RESTORE` behavior using any of the following in the `restore_op
  Option                                                             | <div style="width:75px">Value</div>         | Description
  -------------------------------------------------------------------+---------------+-------------------------------------------------------
 <a name="into_db"></a>`into_db`                                     | Database name                               | Use to [change the target database](#restore-tables-into-a-different-database) for table restores. (Does not apply to database or cluster restores.)<br><br>Example: `WITH into_db = 'newdb'`
```

Comment:
Can you update the description for `into_db` as per DOC-1868? The main ask is to ensure that it is clear in the description that the database needs to exist before running it. Adding the `CREATE DATABASE` statement to the example is a great idea, we also need to allude to this in the option description here. 

So, something like: "The target database must exist before a restore with `into_db`." Or, something to that effect.